tests/unit/Core/Content/Product/Cms/ProductDescriptionReviewsCmsElementResolverTest.php

```diff
@@ -99,13 +156,18 @@ public function testEnrichSetsEmptyDataWithoutConfig(): void
 
     private function getResolver(): ProductDescriptionReviewsCmsElementResolver
     {
+        $this->systemConfigService = new StaticSystemConfigService([
```

Comment:
it is a bit strange to read, that the property is not set up in the `setup` method :grin: